### DISCUSSION TITLE
OServerAdmin Apis were deprecated without providing alternate

### DISCUSSION THREAD

**Discussion by vivekjustthink** [orientechnologies/orientdb#9553] [Q&A] [answered]
Hi, my usecase is to create a database on server cold start using java api. I could use OServerAdmin.createDatabase and  solve the usecase. However, the class seems to be deprecated and alternate for the functionalities were not provided in the code comments. Kindly state if there are specific reasons to it. Here's the commit details (https://github.com/orientechnologies/orientdb/commit/e59ae1e3d8e53b35811793ff970037ab01633e86#diff-fcc7fd9ce7b2a3e92d11c2e5da526383c9ae6656a217a5a3357821137040df69). Thanks for the support.

**Comment by luigidellaquila** [comment, marked as answer]
Hi @vivekjustthink 

OServerAdmin was deprecated because we introduced a more complete and consistent OrientDB class API, that works with the exact same logics in embedded, remote and memory modes.
You can have a quick look at the API from this page https://orientdb.org/docs/3.1.x/java/Document-API-Database.html

Thanks

Luigi

**Reply by vivekjustthink** [reply]
Thanks, this works!
